src/util.ts

```diff
@@ -22,23 +22,9 @@ export function SanitizeMarkdown(str: string): string {
 
 
 export function checkIfMod(id: string): boolean {
-    let isMod = false;
-    Config.moderatorIds.map(val => {
-        if (val === id) {
-            isMod = true;
-        }
-    })
-
-    return isMod;
+  return Config.moderatorIds.some((moderatorId) => moderatorId === id);
```

Comment:
I think array.includes works as well honestly 🤣 

the original code is embarassingly stupid